### ISSUE TITLE
Distribute a 'duotone' font for coding

### ISSUE THREAD

**Issue by iansan5653** [githubnext/monaspace#103]
It would be nice to render italics in a more 'cursive' font than non-italics to make code easier to read. While VSCode doesn't have native support for this, [Recursive Mono](https://www.recursive.design/) achieves it anyway by distributing a "Rec Mono Duotone" font that replaces the italic forms with a slightly more cursive variant of the font:

<img width="255" alt="screenshot of VSCode with rec mono duotone font. Italics look more handwritten than non-italics" src="https://github.com/githubnext/monaspace/assets/2294248/ffd5ed26-5743-4c49-a1bc-0f95077555a8">

Setting the font to "Monaspace Argon" on the other hand results in italics that look more like regular letters but slanted (with a few exceptions like the `f`):

<img width="255" alt="screenshot of VSCode with Monaspace Argon font" src="https://github.com/githubnext/monaspace/assets/2294248/baab0933-c571-432f-ae71-6763eba25766">

This makes the italics harder to distinguish, making code more difficult to read. It would be really cool to try a Argon or Neon but with the italic glyphs replaced with Radon or Xenon.

**Comment by jdraggy** [comment]
Yes, but you do not need to make 'duotone' fonts. Adding that style of italic to the normal font is much better.
